manager/src/main/java/org/openremote/manager/mqtt/UserAssetProvisioningMQTTHandler.java

```diff
@@ -475,8 +526,9 @@ protected X509ProvisioningConfig getMatchingX509ProvisioningConfig(RemotingConne
             .orElse(null);
     }
 
-    public static User getCreateClientServiceUser(String realm, ManagerKeycloakIdentityProvider identityProvider, String uniqueId, ProvisioningConfig<?, ?> provisioningConfig) throws RuntimeException {
-        String username = (PROVISIONING_USER_PREFIX + uniqueId);
+    public static User getCreateClientServiceUser(String realm, ManagerKeycloakIdentityProvider identityProvider, String uniqueId, String userPrefix, ProvisioningConfig<?, ?> provisioningConfig) throws RuntimeException {
+        String prefix = userPrefix != null ? userPrefix : PROVISIONING_USER_PREFIX;
+        String username = (prefix + uniqueId);
```

Comment:
Why are we changing this logic? Useful to be able to identify service users that have been created for provisioning purposes in a consistent way